### ISSUE TITLE
Insufficient color contrast for informative image

### ISSUE THREAD

**Issue by mkdir-washington-edu** [hypothesis/product-backlog#1638]
From the HighWire Press / McGraw-Hill Accessibility spreadsheet: https://hypothes-is.slack.com/archives/C8TPC8XMK/p1736535764817829

**Highwire Press Reproduction**
1. Navigate to the mentioned image button in the main content area.
2. Use any color contrast testing tool to check the color contrast.
3. Observe that the color contrast between the image button and the adjacent background is insufficient.

**HighWire Expected Result**
The contrast ratio between the key part of the image button and its adjacent colors should be sufficient.

**HghWire Actual Result**
The #9C9C9C foreground color of the ""Annotation sidebar"" image button has a insufficient color contrast ratio of 2.7:1 with #FFFFFF background color.

This will make it difficult for people with low vision and color blindness to identify the focus indicator for the user interface controls that have insufficient contrast.

**HighWire Rationale**
All the non text content of a web page including informative images, user interface controls and their different states such as hover, focus, active, etc. should be displayed using sufficient contrast between foreground/background and adjacent colors to make sure that people with low vision and those with color deficiencies can access them.
WCAG 2.1 has outlined the contrast requirement for non text content as 3:1.

**HighWire Recommendation**
For the key parts of all the informative images, ensure that the contrast between foreground, background or adjacent colors is 3:1 to make the image's information accessible for low vision and color-blind users.
